addons/account/models/account_journal.py

```diff
@@ -1129,3 +1159,45 @@ def _process_reference_for_sale_order(self, order_reference):
         '''
         self.ensure_one()
         return order_reference
+
+    # -------------------------------------------------------------------------
+    # E-Invoice Related Methods
+    # -------------------------------------------------------------------------
+
+    def _notify_einvoices_received(self, moves):
+        self.ensure_one()
+
+        if not (einvoice_count := len(moves)):
+            return
+
+        self._send_einvoice_received_mail_if_needed(einvoice_count)
+
+    def _send_einvoice_received_mail_if_needed(self, einvoice_count: int):
```

Comment:
I may be a bit too defensive, but it feels like it would be a better idea to give the moves here.
If tomorrow we want to put other info in the email than the count, we'll have to change the signature which we can't in stable.
This also makes `_notify_einvoices_received` and `_send_einvoice_received_mail_if_needed` redundant, we can probably have only one ? Good thing atm is that we could add other notification "ways" (other than email). I can live with both, I let you decide :wink: